assets/src/components/detours/diversionPage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { DiversionPanel, DiversionPanelProps } from "./diversionPanel"
+import MapDisplay from "../mapPage/mapDisplay"
+
+export const DiversionPage = ({
+  directions,
+  missedStops,
```

Comment:
I think I see this collection of attributes being its own object at some point, like a `Detour` object. Otherwise, we could wind up passing each of these around separate a whole lot, which could get kind of messy.

Reply:
I could justify this change as a refactor in the future, once we see more clearly how these attributes are being used.

Reply:
Defintely agree! So far, I built this to give us as much flexibility over the content and not include any sort of object grouping yet. I'd imagine us creating a `DiversionPageFromDetour` or something of the like which would be the translation from say, a `Detour` object, into the actual properties the base component takes. I think this also gives us more flexibility in Storybook to play around with the content, instead of an object.

Reply:
At some point, I'd at least like to try and see.

<details>
<summary>
Rant
</summary>
In a certain sense, if we have patterns that exist in order to accommodate Storybook, then those patterns are Storybook-only patterns, even if they live outside of the stories/ directory.

In those cases, I'd rather make the rationale explicit by making Storybook do the extra work and putting the extra patterns in Storybook.

But..... if the "extra" pattern also makes the production code easier to work with, then it's not Storybook-only (the location marker thing we paired on a while ago is a good example of that - we made the Storybook integration easier AND made the production code better at the same time)
</details>